bin/plot_krun_results

```diff
@@ -1110,17 +1111,32 @@ def get_data_dictionaries(json_files, benchmarks=[], wallclock_only=False,
         machine_name = pretty_print_machine(machine)
 
         # Is there instrumentation data with this results file?
-        instr_data = False
-        filename_root = filename[:-len('_results.json.bz2')]
-        instr_dir = filename_root + '_instr_data'
-        if 'instr_data' not in data_dictionary:
-            data_dictionary['instr_data'] = dict()
-        if os.path.isdir(instr_dir):
+        # If the input filename is:
+        #     warmup_results_outliers_w200_changepoints.json.bz2
+        # then we expect the VM instrumentation files to be in a directory
+        # called `warmup_instr_data/` If there is more than one data named
+        # *_instr_data/ then we issue a warning to the user and chose one a
+        # random.
```

Comment:
Would an easier approach be to have a `--instr-dir` switch on the command line? Then the user can put the instrumentation data in a directory named as they wish.

Reply:
It's possible. Since @fsfod is our main user at the moment, I'm happy to go with whatever he thinks is sensible. If we do it this way though, I'll probably split the other fixes into a separate PR.

Reply:
Maybe keep it automatic with but the command line an optional override for weird situations

Reply:
As you wish, but let's try to keep it as simple as possible.

Reply:
To be honest, I think we either need to do one or the other. It would be really baffling for a user who doesn't pass in the CLI option to suddenly find that we have picked up data from a directory when they didn't expect it. Any opinions @ltratt ?

Reply:
This does sound like a case where an explicit option is the least surprising thing to do.